### PR TITLE
Story1 show collection

### PR DESCRIPTION
**Pull request by matthewsearle01**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+textarea {
+    width: 100%;
+}
+
+h1.add-recipe-title {
+    text-decoration: underline;
+}
+button {
```
Comment (nealioa) [inline css/styles.css:111]: Separate rules with a blank line.
Also, why is this not in with the button css above (line 73)?

File: includes/tests/functions.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    //formatRecipeName malformed test:
+    public function testRecipeNameMalformed() {
+        //arrange
+        $input = 21;
+        $this->expectException(TypeError::class);
+        //act
+        $result = formatRecipeName($input);
+
```
Comment (nealioa) [inline includes/tests/functions.php:37]: A blank line here is inconsistent with the other functions.

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+button {
+    background-color: #fff0b9;
+    padding: 0.5rem;
+    border: black 3px solid;
+    border-radius: 20px;
+}
+
+.button {
```
Comment (nealioa) [inline css/styles.css:80]: Does your code actually have something with a `class="button"`?

**Review by nealioa — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 1 above)
Comment (matthewsearle01) [reply css/styles.css:111]: Thanks, I've combined this with the button css on line 73

File: includes/tests/functions.php (hunk 2 above)
Comment (matthewsearle01) [reply includes/tests/functions.php:37]: Fixed!

File: css/styles.css (hunk 3 above)
Comment (matthewsearle01) [reply css/styles.css:80]: Yes on index.php line 24

**Pull request closed without merging**